Dockerfile

```diff
@@ -42,13 +42,15 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG BUILD_PACKAGES="\
   build-essential \
   git \
+  mariadb-client \
   python3-dev"
 
 # Packages need for running
 ARG RUNTIME_PACKAGES="\
   curl \
   file \
   # fonts for text file thumbnail generation
+  default-libmysqlclient-dev \
```

Comment:
I know it's minor in the overall image size, but these two packages feel switched to me.  Usually a `-dev` is required for building, while another one is only required for run time.